newsroom/oauth_clients/clients_async.py

```diff
@@ -0,0 +1,53 @@
+import logging
+from pydantic import Field
+from datetime import datetime
+from typing_extensions import Optional, Annotated, List, Dict
+
+
+from superdesk.core.resources import ResourceModel, ResourceConfig, MongoResourceConfig, validators
+from superdesk.core.resources.service import AsyncResourceService
+from superdesk.core.web import EndpointGroup
+
+
+from content_api import MONGO_PREFIX
+
+
+class ClientResource(ResourceModel):
+    name: Annotated[
+        Optional[str],
```

Comment:
`name` is a required field, therefor we should not use `Optional[str]` as it's type